### DISCUSSION TITLE
Comparing reporting options for Amazon Ads

### DISCUSSION THREAD

**Discussion by Karthick2299** [amzn/ads-advanced-tools-docs#178] [Q&A] [answered]
Hi @elizafarley , 

I need information about these three items:

* Reports
* AMS
* AMC

For now, my requirement is to analyse how much we spend on Amazon for ads, and I also need historical ad data if possible.

Currently, I'm obtaining ad spend data from the `Advertised Product Report`. I can access yesterday's advertised product data and then store it in a database with the help of AWS Lambda.

In `AMS`, we can obtain hourly-based ad spend data and store it in the database.

I am currently exploring `AMC` and need to learn more about it.

My questions are:
1. Among these three options, which one is the best for us to obtain ad spend data on Amazon?
2. Can you provide me with the pros and cons of each of these three options?

Thanks in advance.

**Comment by kdwivedi13** [comment, marked as answer]
Reporting API

* Use case: Provides daily grain reporting data for Amazon DSP, Sponsored Brands, Sponsored Products, and Sponsored Display.
* Benefits: Enables improved campaign optimization through daily reporting.
* Eligibility: Open to all Amazon advertisers, tool providers, and agencies.
* Supported products: Amazon DSP, Sponsored Brands, Sponsored Products, and Sponsored Display.
* Data frequency:  Data is available up to that point in time, but may be inaccurate due to a daily reconciliation.
* Data granularity: Daily 
* Interface: API
* Historical Data -  Varies, but up to 95 days
* Primary purpose: Reporting, analysis, and planning
* Customization: No customization of input data and metrics.
* Cost: No cost.

AMC

* Use cases:
    * Performance deep dive: Analyze a specific campaign to uncover insights and identify areas for improvement
    * Audience insights: Discover correlations between customer engagement and attributes to identify your optimal audience segment
    * Journey analysis: Illustrate the customer shopping journey, including media exposure, frequency, timing, and other relevant factors.
    * Media mix analysis: Understand the collective impact of different media channels and isolate the incremental impact of individual channels.
    * Omni-channel impact analysis: Assess Amazon media's contribution to sales across various Amazon store types and your D2C channels.
    * Custom attribution: Allocate credit to different touchpoints on the customer journey
* Benefits: Data clean room for event-level metrics enables custom analytics in a privacy-safe environment.
* Eligibility: Open to Amazon DSP and Sizmek Ad Suite advertisers, tool providers, and agencies.
* Supported products: Amazon DSP, Sizmek Ad Suite, and sponsored ads.
* Data frequency: 6-7 hour delay for traffic; 24hr-28 days for conversion.
* Data granularity: Event level data.
* Interface: UI and API. Both allow users to submit SQL queries.
* Historical Data -  Up to 7 days 
* Primary purpose: Analytics.
* Customization: High customization of input data and metrics. Users can also upload their own datasets into AMC, providing for more use cases over custom data in addition to the Amazon-loaded data.
* Cost: No cost, but must be an Amazon DSP or Sizmek Ad Suite advertiser.

Amazon Marketing Stream


* Use cases: Provides reporting and insights, on non-PII hourly data to enable intraday campaign management and near real-time decision making.
* Benefits: Information is pushed/streamed instead of being pulled, providing hourly cadence for reporting metrics and near real-time messaging.
* Eligibility: Open to all Amazon advertisers, tool providers, and agencies.
* Supported products: Sponsored ads, Amazon DSP
* Data frequency: 1-hour delay for traffic; conversion at 1 hour, 24 hours, and 7 days.
* Data granularity: Hourly.
* Interface: Subscription API for integrators, AWS tools for S3 data lake.
* Primary purpose: Reporting, analysis.
* Customization: No customization.
* Cost: Cost for AWS SQS Queue and Compute resource, if used.


The best service for you will depend on your specific needs. If you need daily reporting to help with campaign optimization and media planning, then the reporting API is a good choice. If you need to perform custom analytics or attribution, then AMC is a good option. If you need intraday campaign management or near real-time decision making, then Amazon Marketing Stream is a good choice. You can also use multiple services in combination to meet your needs. For example, you could use the reporting API for daily reporting and Amazon Marketing Stream for intraday campaign management. Or, you could use AMC for custom analytics and Amazon Marketing Stream API for near real-time decision making.

I hope this information is helpful. Please let me know if you have any other questions

**Reply by Karthick2299** [reply]
Thanks @elizafarley and @kdwivedi13. 
For your Valuable Response.
